src/resources/postcss/views/skeleton/_single-event.pcss

```diff
@@ -344,3 +344,51 @@
 	}
 }
 
+/* Footer Navigation */
+#tribe-events-footer {
+	padding: var(--spacer-4) 0;
+	
+	@media (--viewport-medium) {
+		padding: var(--spacer-6) 0;
+	}
+}
+
+.tribe-events-single .tribe-events-sub-nav {
+	display: flex;
+	justify-content: space-between;
+	
+	a {
+		&:before,
+		&:after {
+			content: '';
+			display: inline-block;
+			height: 20px;
+			vertical-align: middle;
+			width: 11px;
+		}
```

Comment:
this stuff we should just put straight into full styles as it doesn't make any sense without the background image.

Reply:
will do, thanks